clang/tools/dpct/cmake/dpct.cmake

```diff
@@ -78,3 +78,24 @@ endmacro()
 # Always set SYCL_HAS_FP16 to true to assume SYCL device to support float16
 message("dpct.cmake: SYCL_HAS_FP16 is set true by default.")
 set(SYCL_HAS_FP16 TRUE)
+
+# Find oneAPI lib packages
+macro(DPCT_FIND_PACKAGE package)
+  DPCT_GET_SOURCES(_components ${ARGN})
+
+  foreach(package_name _components)
+    if(package_name STREQUAL "cublas" OR
+      package_name STREQUAL "curand" OR
+      package_name STREQUAL "sparse" OR
```

Comment:
Pls make sure the cuda libraries are valid arguments for find_package in cuda side?